README.md

```diff
@@ -56,6 +56,7 @@ General attributes:
 * `node['splunk']['web_port']`: The port that the splunkweb service
   listens to. This is set to the default for HTTPS, 443, as it is
   configured by the `setup_ssl` recipe.
+* `node['splunk']['ratelimit_kilobytessec']`: The default splunk rate limiting rate can now easily be changed with an attribute.  Default is 2048KBytes/sec.
```

Comment:
Perhaps as a future consideration, name this more like the actual setting? E.g., `node['splunk']['limits']['maxKBps']`?